utilities/audio_utils.py

```
@@ -91,3 +96,48 @@ async def process_prefix_audio(prefix_audio_path: str, model, device: torch.devi
 
     return audio_prefix_codes
+
+
+def init_latent_cache() -> None:
+    """Initialize latent cache from disk for all supported languages."""
+    get_embed_cache_dir.cache_clear()
+
+    cache_manager = get_embed_cache_manager()
+    cache_manager.clear_memory()
+
+    cached_latents = {}
```

Ruff (0.14.1)

Local variable `cached_latents` is assigned to but never used

Remove assignment to unused variable `cached_latents`

(F841)